CodeStarterCamp_Week1/Method.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Method.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by 장경빈 on 2022/04/18.
+//
+
+import Foundation
+
+
+func drawPepero(bodyLength: Int, body : String, topping : String = " ",  stickLength : Int){
+    print("<정보>")
+    print("길이: \(bodyLength)")
+    print("몸통: \(body)")
+    print("토핑: \(topping)")
+    print("막대길이: \(stickLength)\n")
+    
+    drawBody(body,topping,bodyLength)
+    drawStick(stickLength)
+    print()
+}
+
+func drawStick(_ length: Int){
+    for _ in 1...length{
+        print(" | |")
+    }
+}
+
+func drawBody(_ body: String, _ topping : String, _ length: Int){
+    for i in 1...length{
+        if i%2 == 1{
+            print("\(topping)\(body)")
+        }
+        else{print(" \(body)\(topping)")
+        }
+        
+    }
+}
+
```

Comment:
오.. 이런 방식으로 해결하셨군요.
for 문의 `임시 상수(i)`가 1부터 순서대로 오를 때, 짝/홀을 조건문으로 판단해서 print 를 출력해주는 거군요!
아주 잘 하셨습니다. 👏🏻👏🏻

다만, 코드 작성하실 때 줄바꿈이나 스페이스바 여백을 줄 수 있도록
커밋하기 전에 한 번 점검해보시는 건 어떨까요?

뭔가 바둑이님의 코드를 읽다 보면, 시간에 쫓겨서 급하게 코드를 작성한 느낌이 납니다..! 😢 
코드의 난이도와 관계 없이, `규칙적이고 읽기 편한 코드`를 쓸 수 있도록 신경쓰신다면, 훨~~씬 좋겠습니다! 😄 

```suggestion
func drawBody(_ body: String, _ topping : String, _ length: Int) {
    for i in 1...length {
        if i % 2 == 1 {
            print("\(topping)\(body)")
        } else {
            print(" \(body)\(topping)")
        }
    }
}
```